src/views/Settings.vue

```diff
@@ -309,10 +330,35 @@ export default defineComponent({
         }
       } catch(err) {
         logger.error('Failed to fetch current facility details', err)
-      } finally {
-        // declaration of isStoreFulfilmentTurnedOn in lifecycle hooks always
-        // returns 'true' because of the != 0 condition, hence, updating it here
-        this.isStoreFulfilmentTurnedOn = this.currentFacilityDetails?.maximumOrderLimit != 0
+      }
+    },
+    async getFacilityOrderCount() {
+      let resp: any;
+      try {
+        resp = await UserService.getCurrentOrdersCount({
+          "entityName": "FacilityOrderCount",
+          "inputFields": {
+            "facilityId": this.currentFacility.facilityId
+          }
```

Comment:
When calling performFind, we need to define viewSize, fieldList to get specific data from api